Kentico.Kontent.Management/Models/Shared/ListingResponseModel.cs

```diff
@@ -5,30 +5,23 @@
 
 namespace Kentico.Kontent.Management.Models.Shared
 {
-    /// <summary>
-    /// Represents listing response.
-    /// </summary>
-    public class ListingResponseModel<T> : IEnumerable<T>
```

Comment:
I think the interface needs to inherit IEnumerable<T>, this way you want to be able to access items if you mock IListingResponseModel.